theseus/src/launcher/mod.rs

```diff
@@ -462,27 +462,24 @@ pub async fn launch_minecraft(
     use regex::Regex;
 
     let options_path = instance_path.join("options.txt");
-    let mut options_string = String::new();
-
     if options_path.exists() {
-        options_string = io::read_to_string(&options_path).await?;
-    }
-
-    for (key, value) in mc_set_options {
-        let re = Regex::new(&format!(r"(?m)^{}:.*$", regex::escape(key)))?;
-        // check if the regex exists in the file
-        if !re.is_match(&options_string) {
-            // The key was not found in the file, so append it
-            options_string.push_str(&format!("\n{}:{}", key, value));
-        } else {
-            let replaced_string = re
-                .replace_all(&options_string, &format!("{}:{}", key, value))
-                .to_string();
-            options_string = replaced_string;
+        let mut options_string = io::read_to_string(&options_path).await?;
+        for (key, value) in mc_set_options {
+            let re = Regex::new(&format!(r"(?m)^{}:.*$", regex::escape(key)))?;
+            // check if the regex exists in the file
+            if !re.is_match(&options_string) {
+                // The key was not found in the file, so append it
+                options_string.push_str(&format!("\n{}:{}", key, value));
+            } else {
+                let replaced_string = re
+                    .replace_all(&options_string, &format!("{}:{}", key, value))
+                    .to_string();
+                options_string = replaced_string;
+            }
         }
```

Comment:
if someone has an existing options.txt, it should add to it, not override it

Reply:
Fixed something in it separate to this comment.

currently if an options.txt exists, this loads it, replaces relevant values, adds relevant new ones, and writes back to file. It doesn't override the file, only the chosen options and writes it back. Unless you mean it shouldn't overwrite the option but should add an additional one (ie: two entries for fullscreen) in which case I may be misunderstanding